content/docs/reference/pack-cli/_index.md

```diff
@@ -0,0 +1,6 @@
++++
+title="Pack Command Line Tool"
```

Comment:
From a marketing perspective this should be Pack CLI for consistency and simply sake.

```suggestion
title="Pack CLI"
```

Reply:
Ok. I had changed it due to the feedback from  buildpacks/pack#812, but I also find CLI a bit simpler